README.md

```diff
@@ -234,6 +234,32 @@ plugins you don't care about). Use the --help option when configuring
 for more info.
 
 
+## Compile on macOS
+
+### Install dependencies
+
+```
+brew install jansson libnice openssl libusrsctp libmicrohttpd libwebsockets cmake rabbitmq-c sofia-sip opus libogg glib pkg-config gengetopt
+```
+
+```
+wget https://github.com/cisco/libsrtp/archive/v1.5.4.tar.gz
+tar xvf v1.5.4.tar.gz
+cd libsrtp-1.5.4
+./configure --prefix=/usr/local/libsrtp
+make
+sudo make install
+```
+
+### Build janus gateway
+
+```
+cd /path/to/janus-gateway
+./configure --disable-unix-sockets --disable-rabbitmq --disable-mqtt --prefix=/usr/local/janus PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig
```

Comment:
unix-sockets is not supported on macOS.

OK, I will remove the other 2 disable options